### PR TITLE
STREAMS-1705 Add release note section

### PR DESCRIPTION
**Pull request by allandenis38**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

* Add release note section to Streams documentation
* Add release note for Streams Jan21 release

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

**Comment by allandenis38** [comment]
@lbadenhop PR ready for review.
Note: Merge to master is intended.

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Streams Jan 2021 Release Notes
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:2]:
```suggestion
title: Streams January 2021 Release Notes
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+linkTitle: Streams Jan 2021 Release Notes
+weight: 150
+date: 2021-02-03
+---
+
+## Summary
+
+Streams is available as a set of Docker containers deployable in Kubernetes thanks to a Helm Chart.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:10]:
```suggestion
Streams is available as a set of Docker containers deployable in Kubernetes by using a Helm Chart.
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+* [Webhook Testing Endpoint](/docs/subscribers/subscriber-webhook/#testing-a-webhook-subscription) to enable API consumers to send test payloads to their webhook endpoint.
+
+## Important changes
+<!-- Use this section to describe any changes in the behavior of the product (as a result of features or fixes), for example, new Java system properties in the jvm.xml file. This section could also be used for any important information that doesn't fit elsewhere. -->
+
+It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this new version.
+
+### Webhook subscription status
+
+The Webhook Subscription status `subscriptionStatus` is now automatically set to `suspended` when the Webhook endpoint responds with `410 GONE` status code. When subscription is status is `suspended`, Streams no longer attempts to send Webhook notifications. The subscription can be reactivated by setting `subscriptionStatus` to `active` via a `PATCH` operation on `/subscribers/webhook/subscriptions/{{subscriptionId}}` endpoint.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:25]:
```suggestion
The webhook subscription status `subscriptionStatus` is now automatically set to `suspended` when the webhook endpoint responds with `410 GONE` status code. When subscription status is `suspended`, Streams no longer attempts to send webhook notifications. The subscription can be reactivated by setting `subscriptionStatus` to `active` via a `PATCH` operation on `/subscribers/webhook/subscriptions/{{subscriptionId}}` endpoint.
```

Comment (lbadenhop) [reply content/en/docs/relnotes/20210127_streams_relnotes.md:25]: webhook vs Webhook, subscription vs Subscription inconsistencies in this section and in previous sections

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+---
+
+## Summary
+
+Streams is available as a set of Docker containers deployable in Kubernetes thanks to a Helm Chart.
+For a summary of the system requirements, see [Install Streams](/docs/install/).
+
+## New features and enhancements
+<!-- Add the new features here -->
+* [Webhook Exchange History](/docs/subscribers/subscriber-webhook/#getting-the-webhook-notification-history-for-a-subscription) to give API consumers access to the history of exchanges (requests/responses) that occurred between Streams and their Webhook endpoint.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:15]: webhook or Webhook?

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+* [Webhook Testing Endpoint](/docs/subscribers/subscriber-webhook/#testing-a-webhook-subscription) to enable API consumers to send test payloads to their webhook endpoint.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:16]: webhook or webhook?

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+<!-- Use this section to describe any changes in the behavior of the product (as a result of features or fixes), for example, new Java system properties in the jvm.xml file. This section could also be used for any important information that doesn't fit elsewhere. -->
+
+It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this new version.
+
+### Webhook subscription status
+
+The Webhook Subscription status `subscriptionStatus` is now automatically set to `suspended` when the Webhook endpoint responds with `410 GONE` status code. When subscription is status is `suspended`, Streams no longer attempts to send Webhook notifications. The subscription can be reactivated by setting `subscriptionStatus` to `active` via a `PATCH` operation on `/subscribers/webhook/subscriptions/{{subscriptionId}}` endpoint.
+
+### Streams Helm Chart enhancements
+
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:28]: Helm Chart, helm chart, or Helm chart? Inconsistent throughout section.

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+Following enhancements have been made to Streams Helm Chart:
+
+* Helm chart dependency images upgraded:
+    * Kafka new docker image tag: `2.6.0-debian-10-r110`
+    * Zookeeper new docker image tag: `3.6.2-debian-10-r112`
+    * MariaDB new docker image tag: `10.4.17`
+    * Nginx new docker image tag: `v0.43.0`
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:35]:
```suggestion
    * NGINX new docker image tag: `v0.43.0`
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+* Helm chart dependencies upgraded:
+    * Nginx helm chart new version: `3.20.1`
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:38]:
```suggestion
    * NGINX helm chart new version: `3.20.1`
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this new version.
+
+### Webhook subscription status
+
+The Webhook Subscription status `subscriptionStatus` is now automatically set to `suspended` when the Webhook endpoint responds with `410 GONE` status code. When subscription is status is `suspended`, Streams no longer attempts to send Webhook notifications. The subscription can be reactivated by setting `subscriptionStatus` to `active` via a `PATCH` operation on `/subscribers/webhook/subscriptions/{{subscriptionId}}` endpoint.
+
+### Streams Helm Chart enhancements
+
+Following enhancements have been made to Streams Helm Chart:
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:29]:
```suggestion
The following enhancements have been made to Streams Helm Chart:
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+* Helm chart refactoring:
+    * All common microservice parameters can be set in a single place.
+    * `AdditionalJavaOpts` has been added.
+    * `JvmMemoryOpts` will be computed automatically from k8s resources if not specified.
+    * `ingress.host` helm chart parameter is now mandatory at installation. See [Ingress hostname](/docs/install/#ingress-hostname) section for details.
+
+## Deprecated features
+<!-- Add features that are deprecated here -->
+
+As part of our software development life cycle we constantly review our Streams offering.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:49]:
```suggestion
As part of our software development life cycle, we constantly review our Streams offering.
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+As part of this review, no capabilities have been deprecated.
+
+## Removed features
+<!-- Add features that are removed here -->
+
+To stay current and align our offerings with customer demand and best practices, Axway might discontinue support for some capabilities. As part of this review, no capabilities have been removed.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:55]:
```suggestion
To stay current and align our offerings with customer demand and best practices, Axway might discontinue support for some capabilities. No capabilities have been removed in this release.
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+
+## Fixed issues
+
+There are no fixed known issue in this version.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:59]:
```suggestion
There are no fixed issue in this version.
```

Comment (lbadenhop) [reply content/en/docs/relnotes/20210127_streams_relnotes.md:59]: Are there any known issues in this release?

File: content/en/docs/relnotes/20210127_streams_relnotes.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+To find all available documents for this product version:
+
+1. Go to <https://docs.axway.com/bundle>.
+2. In the left pane Filters list, select your product or product version.
+
+Customers with active support contracts need to log in to access restricted content.
+
+The following reference documents are also available:
+
+* [Streams Security guide](https://docs.axway.com/bundle/Streams_20_SecurityGuide_allOS_en_HTML5/)
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210127_streams_relnotes.md:90]: The Security Guide is restricted access, so I wouldn't add a link to the guide.

**Review by lbadenhop — COMMENTED** [review]
@allandenis38 See suggestions.

File: content/en/docs/relnotes/20210127_streams_relnotes.md (hunk 4 above)
Comment (allandenis38) [reply content/en/docs/relnotes/20210127_streams_relnotes.md:15]:
```suggestion
* [Webhook Exchange History](/docs/subscribers/subscriber-webhook/#getting-the-webhook-notification-history-for-a-subscription) to give API consumers access to the history of exchanges (requests/responses) that occurred between Streams and their webhook endpoint.
```

File: content/en/docs/relnotes/20210127_streams_relnotes.md (hunk 12 above)
Comment (allandenis38) [reply content/en/docs/relnotes/20210127_streams_relnotes.md:59]: I confirm that we do not have any known customer defect at the moment.

File: content/en/docs/relnotes/20210127_streams_relnotes.md (hunk 13 above)
Comment (allandenis38) [reply content/en/docs/relnotes/20210127_streams_relnotes.md]: I've removed the links.

**Pull request merged**